src/network/mcpe/handler/InGamePacketHandler.php

```diff
@@ -736,7 +737,9 @@ public function handleItemFrameDropItem(ItemFrameDropItemPacket $packet) : bool{
 		$blockPosition = $packet->blockPosition;
 		$block = $this->player->getWorld()->getBlockAt($blockPosition->getX(), $blockPosition->getY(), $blockPosition->getZ());
 		if($block instanceof ItemFrame && $block->getFramedItem() !== null){
-			return $this->player->attackBlock(new Vector3($blockPosition->getX(), $blockPosition->getY(), $blockPosition->getZ()), $block->getFacing());
+			$this->player->getWorld()->addSound($pos = new Vector3($blockPosition->getX(), $blockPosition->getY(), $blockPosition->getZ()), new ItemFrameRemoveItemSound());
```

Comment:
This should be in `ItemFrame->onAttack()`, you don't know if the item frame is empty

Reply:
Now that I see that code above it it does check, but you don't know if a plugin has canceled the interact action